torchrl/envs/libs/pettingzoo.py

```diff
@@ -0,0 +1,929 @@
+import copy
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+from tensordict.tensordict import TensorDictBase
+
+from torchrl.data import (
+    CompositeSpec,
+    DiscreteTensorSpec,
+    OneHotDiscreteTensorSpec,
+    UnboundedContinuousTensorSpec,
+)
+from torchrl.envs.common import _EnvWrapper
+from torchrl.envs.libs.gym import _gym_to_torchrl_spec_transform, set_gym_backend
+from torchrl.envs.utils import _check_marl_grouping, _replace_last, MarlGroupMapType
+
+IMPORT_ERR = None
+try:
+    import pettingzoo
+
+    _has_pettingzoo = True
+
+except ImportError as err:
+    _has_pettingzoo = False
+    IMPORT_ERR = err
+
+__all__ = ["PettingZooWrapper", "PettingZooEnv"]
```

Comment:
we should move away from `__all__` as `from .pettingzoo import *` will not be used.